### ISSUE TITLE
Can't log in on the browser of BMBF

### ISSUE THREAD

**Issue by troubledboy** [kihecido/BMBF#69]
HI i'm a new user, 

The install all went fine as far as i can tell but when i try and log in to bsaber using the launcher it accepts my password but doesn't actually log me in?

I can log in on my browser (on my laptop) without issue...

**Comment by stuart20202** [comment]
im having the same problem it wont let me long in on the headset

**Comment by Steveorevo** [comment]
Yup. It's broken. Acting like it can't record the session/cookie and just redisplays the home page with the login menu option (it should read "Logout"). Definitely not logging in. However I have no issue with login on desktop/smartphone.

**Comment by micfogas** [comment]
so is the issue related to the inbuilt browser of bmbf or a change made by bsaber.com? does beatsaver cause the same problems? I had an issue before where the instant install didn't work but choosing the download option would automatically install it. I haven't updated yet. can anyone confirm whether the download option works for installing the song? does the problem happen when connecting to bmbf web interface from a desktop or mobile device using the IP:port displayed on the tools tab, or does it only happen when accessing bmbf interface on the quest?




this detailed level of info is useful for helping the devs determine the cause of the problem, which is necessary before it can be fixed.






On Wed, Apr 8, 2020 at 1:16 AM -0700, "Stephen J. Carnam" <notifications@github.com> wrote:















Yup. It's broken. Acting like it can't record the session/cookie and just redisplays the home page with the login menu option (it should read "Logout"). Definitely not logging in. However I have no issue with login on desktop/smartphone.
